apps/common/permissions.py

```diff
@@ -68,3 +68,9 @@ def has_permission(self, request, view):
 
     def has_object_permission(self, request, view, obj):
         return False
+
+
+class IsValidLicense(permissions.BasePermission):
+
+    def has_permission(self, request, view):
+        return settings.XPACK_LICENSE_IS_VALID
```

Comment:
There were no significant differences found between the two codes. They contain identical functions and conditions.